sources/api/pluto/src/main.rs

```diff
@@ -383,3 +455,101 @@ mod test_pause_container_account {
         );
     }
 }
+
+#[cfg(test)]
+mod test_get_cpu_millicores_to_reserve {
+    use std::collections::HashMap;
+    use crate::get_cpu_millicores_to_reserve;
+    #[test]
+    fn get_cpu_millicores_to_reserve_ok() {
+        let mut cpu_reserved: HashMap<usize, &str> = HashMap::new();
+        cpu_reserved.insert(0, "0m");
+        cpu_reserved.insert(1, "60m");
+        cpu_reserved.insert(2, "70m");
+        cpu_reserved.insert(3, "75m");
+        cpu_reserved.insert(4, "80m");
+        cpu_reserved.insert(5, "82.5m");
```

Comment:
Fractional millicores don't make a ton of sense to me. Even if kubelet accepts values like this - does it? - we should round up or down to an integer value.

Reply:
Yeah. `kubelet` accepts  fractional millicores like `82.5m` and I will round down. Can you explain why fractional millicores don't make sense?